cmd/frakti/frakti.go

```diff
@@ -41,7 +41,7 @@ const (
 )
 
 var (
-	version = pflag.Bool("version", false, "Print version and exit")
+	version = pflag.Bool("versions", false, "Print version and exit")
```

Comment:
why change it to versions?

Reply:
Avoid errors: "/usr/bin/frakti flag redefined: version"

Reply:
This is ugly, please remove `version` and print out frakti version whenever it started with:
```
glog.Infof("frakti version: %s", fraktiVersion)
```

Reply:
ping @liyanyanli :)